BAZProject/MovieDetail/MovieDetailViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MovieDetailViewController.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MovieDetailViewController: UIViewController, MovieDetailViewProtocol {
```

Comment:
No olvides el final en todas las clases
```suggestion
final class MovieDetailViewController: UIViewController, MovieDetailViewProtocol {
```